src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/persist/SQLParsingUtil.java

```diff
@@ -68,11 +75,11 @@ private static Record getRecordByName(Result<Record> result, Field<String> field
    * @param dataField The field to retrieve data from
    * @return the data read from SQL result
    */
-  public static double readDataFromSqlResult(Result<Record> result, Field<String> matchedField, String matchedFieldName, String dataField) {
+  public double readDataFromSqlResult(Result<Record> result, Field<String> matchedField, String matchedFieldName, String dataField) {
     double ret = Double.NaN;
     if (result != null) {
       try {
-        Record record = SQLParsingUtil.getRecordByName(result, matchedField, matchedFieldName);
+        Record record = getRecordByName(result, matchedField, matchedFieldName);
```

Comment:
`getRecordByName()` is marked `VisibleForTesting` but this caller is not. When the application (and not the tests) calls it, it will result in error, right ?

Reply:
The @VisibleForTesting annotation just indicates that an element is only used for testing or has wider scope than is strictly necessary to allow for testing. I changed the method signature to be package private so that it could be tested, which is why I included the annotation. The annotation doesn't actually do anything at runtime.

Reply:
Removed the annotation, returned to static, non-singleton